tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml

```diff
@@ -78,18 +78,28 @@
         </test>
     </tests>
     <help>
+
+.. class:: warningmark
+
+The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
+automatically be converted to binary before they are analyzed.  This step is performed using the
+ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
+**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
+first manually convert the image to binary using the **Convert to binary (black and white) with
```

Comment:
@bgruening Thanks - when I saw the wrappers in the Galaxy core I didn't check elsewhere.  @davebx @blankenberg I suppose sine this particular wrapper is outside of the Galaxy core, it can remain here and not be moved to tools-iic, but let me know.  Also, @davebx let me know how this and the meme wrapper from the Galaxy core can be made available in the TTS and main TS.